core/src/com/unciv/models/ruleset/IConstruction.kt

```diff
@@ -141,12 +141,14 @@ class RejectionReason(val type: RejectionReasonType,
         RejectionReasonType.MaxNumberBuildable,
     )
     private val orderedImportantRejectionTypes = listOf(
+        RejectionReasonType.ShouldNotBeDisplayed,
         RejectionReasonType.WonderBeingBuiltElsewhere,
         RejectionReasonType.NationalWonderBeingBuiltElsewhere,
         RejectionReasonType.RequiresBuildingInAllCities,
         RejectionReasonType.RequiresBuildingInThisCity,
         RejectionReasonType.RequiresBuildingInSomeCity,
         RejectionReasonType.RequiresBuildingInSomeCities,
+        RejectionReasonType.CanOnlyBeBuiltInSpecificCities,
```

Comment:
Please add this to `constructionRejectionReasonType` as well, since units will now be making use of it

Reply:
Good catch. Done